CITATION.cff

```diff
@@ -0,0 +1,31 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: >-
+  Ivy: Templated deep learning for inter-framework
+  portability
+message: >-
+  If you are going to use this, please cite it by using this
+  message, and we really appreciate your support.
+type: software
```

Comment:
do you think this is the optimal type tag? technically what they are citing is a paper :thinking:

Reply:
yes, I wanted to check if we need to have it as paper or software, I wanted to know what we expect since there is a few possible options, but I will change it for paper right now.: 
![image](https://github.com/unifyai/ivy/assets/18273833/440a49f8-691c-4860-898e-9639ce60b725)